projects/knora-ui/src/lib/viewer/values/date-value/date-value.component.ts

```diff
@@ -0,0 +1,198 @@
+import {Component, Inject, Input, OnChanges, OnDestroy, OnInit, SimpleChanges, ViewChild} from '@angular/core';
+import {CreateDateValue, KnoraDate, KnoraPeriod, ReadDateValue, UpdateDateValue} from '@knora/api';
+import {
+  AbstractControl,
+  FormBuilder,
+  FormControl,
+  FormGroup,
+  FormGroupDirective,
+  NgForm,
+  ValidatorFn
+} from '@angular/forms';
+import {Subscription} from 'rxjs';
+import {BaseValueComponent} from '../base-value.component';
+import {ErrorStateMatcher} from '@angular/material';
+import {DateInputComponent} from "./date-input/date-input.component";
+
+/** Error when invalid control is dirty, touched, or submitted. */
+export class IntervalErrorStateMatcher implements ErrorStateMatcher {
```

Comment:
class name should be adapted to "DateErrorStateMatcher"

Reply:
done in https://github.com/dasch-swiss/knora-ui-ng-lib/pull/35/commits/f5d7a65a7ef47cfd5d8eb6f7181a70e179325197